internal/drivers/anka/pool.go

```diff
@@ -0,0 +1,237 @@
+package anka
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/drone-runners/drone-runner-aws/internal/lehelper"
+	"github.com/drone-runners/drone-runner-aws/types"
+	"github.com/drone/runner-go/logger"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	ankabin = "/usr/local/bin/anka"
+)
+
+type ankaShow struct {
+	UUID    string `json:"uuid"`
+	Name    string `json:"name"`
+	Created string `json:"created"`
+	Memory  string `json:"memory"`
+	IP      string `json:"ip"`
+}
+
+func (p *provider) RootDir() string {
+	return p.rootDir
+}
+
+func (p *provider) ProviderName() string {
+	return string(types.ProviderAnka)
+}
+
+func (p *provider) Ping(_ context.Context) error {
+	return nil
+}
+
+func (p *provider) CanHibernate() bool {
+	return false
+}
+
+func (p *provider) Create(ctx context.Context, opts *types.InstanceCreateOpts) (instance *types.Instance, err error) {
```

Comment:
there is little or no context given in error messages when there is a failure in this method, we just return err. 
it would be helpful to explain what we were trying to do when we failed.

Reply:
added additional logging